README.md

```diff
@@ -767,169 +1206,248 @@ The description is an an optional string for a human-readable description of thi
+
+Valid options: `true`, `false`, `present`, `absent`.
+
+Defaults value: `present`
 
 ##### `resource_name`
 
-The name of the Resource to be created, relative to the java:comp/env context. Default: `$name`
+The name of the Resource to be created, relative to the java:comp/env context.
+
+Default value: `$name`
 
 ##### `resource_type`
 
 The fully qualified Java class name expected by the web application when it performs a lookup for this resource. Required to create the resource.
 
 ##### `catalina_base`
 
-Specifies the root of the Tomcat installation. Default: `$tomcat::catalina_home`
+Specifies the root of the Tomcat installation.
+
+Default value: `$tomcat::catalina_home`
 
 ##### `additional_attributes`
 
-Specifies any additional attributes to add to the Valve. Should be a hash of the format 'attribute' => 'value'. This parameter is optional.
+Specifies any additional attributes to add to the Valve. Should be a hash of the format 'attribute' => 'value'. 
+
+Optional
 
 ##### `attributes_to_remove`
 
-Specifies any attributes to remove from the Valve. Should be a hash of the format 'attribute' => 'value'. This parameter is optional.
+Specifies any attributes to remove from the Valve. Should be a hash of the format 'attribute' => 'value'.
+
+Optional
 
 #### tomcat::config::context::resourcelink
 Specifies a ResourceLink element in the designated xml configuration.
 
 ##### `ensure`
 
-specifies whether you are trying to add or remove the ResourceLink element. Valid values are 'true', 'false', 'present', and 'absent'. Defaults to 'present'
+specifies whether you are trying to add or remove the ResourceLink element.
+
+Valid options: `true`, `false`, `present`, and `absent`.
+
+Default value: `present`
 
 ##### `catalina_base`
 
-Specifies the root of the Tomcat installation. Default: `$tomcat::catalina_home`
+Specifies the root of the Tomcat installation.
+
+Default value: `$tomcat::catalina_home`
 
 ##### `resourcelink_name`
 
-The name of the ResourceLink to be created, relative to the java:comp/env context. Default: `$name`
+The name of the ResourceLink to be created, relative to the java:comp/env context.
+
+Default value: `$name`
 
 ##### `resourcelink_type`
 
 The fully qualified Java class name expected by the web application when it performs a lookup for this resource link.
 
 ##### `additional_attributes`
 
-Specifies any additional attributes to add to the Valve. Should be a hash of the format 'attribute' => 'value'. This parameter is optional.
+Specifies any additional attributes to add to the Valve. Should be a hash of the format 'attribute' => 'value'.
+
+Optional
 
 ##### `attributes_to_remove`
 
-Specifies any attributes to remove from the Valve. Should be a hash of the format 'attribute' => 'value'. This parameter is optional.
+Specifies any attributes to remove from the Valve. Should be a hash of the format 'attribute' => 'value'.
+
+Optional
 
 #### tomcat::install
```

Comment:
Add a space in between this title and the next sentence, this needs to be continued from here to the end of the doc.